library/modules/Textures.cpp

```diff
@@ -120,7 +266,12 @@ static size_t load_textures(color_ostream & out, const char * fname,
 // texture count so our textures will no longer be freed when worlds are
 // unloaded.
 //
-void Textures::init(color_ostream &out) {
+void Textures::init(color_ostream& out) {
```

Comment:
with the interposed handlers in place, we no longer need to call `Textures::init(out)` from Core on every screen transition. You can modify Core.cpp to only call `Textures::init` once.

Reply:
Then it would be something like new `Texture::dynamicInit()`, because we still need to handle static asstest case, right?